.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateParser.cs

```diff
@@ -572,15 +576,7 @@ private DateTimeResolutionResult ParseWeekdayOfMonth(string text, DateObject ref
             var noYear = false;
             int year;
 
-            int cardinal;
-            if (this.config.IsCardinalLast(cardinalStr))
-            {
-                cardinal = 5;
-            }
-            else
-            {
-                cardinal = this.config.CardinalMap[cardinalStr];
-            }
+            var cardinal = this.config.IsCardinalLast(cardinalStr) ? 5 : this.config.CardinalMap[cardinalStr];
```

Comment:
If we declare a constant for the meaning of 5 here, wouldn't it make the code easier to read?